### DISCUSSION TITLE
Other Browsing Data Cleanup Options empty in Firefox 100.2

### DISCUSSION THREAD

**Discussion by chrcoluk** [Cookie-AutoDelete/Cookie-AutoDelete#1396] [Q&A / Support] [answered]
I noticed very recently all of my expressions for whitelisted domains only have the option of keeping all cookies.  All other options dont appear.

In addition the "Other Browsing Data Cleanup Options" has nothing below it in the main settings so is now effectively an empty section.

I am not sure whats going on here, is it meant to be like this on firefox?  No ability to restrict html5 storage?

**Comment by kennethtran93** [comment, marked as answer]
Please update to CAD 3.7.0 which will fix the issue of the missing browsing data items.
